specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2018-06-01-preview/applications.json

```diff
@@ -0,0 +1,403 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "HDInsightManagementClient",
+        "description": "The HDInsight Management Client.",
+        "version": "2015-03-01-preview"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "security": [{
+        "azure_auth": [
+            "user_impersonation"
+        ]
+    }],
+    "securityDefinitions": {
+        "azure_auth": {
+            "type": "oauth2",
+            "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+            "flow": "implicit",
+            "description": "Azure Active Directory OAuth2 Flow",
+            "scopes": {
+                "user_impersonation": "impersonate your user account"
+            }
+        }
+    },
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HDInsight/clusters/{clusterName}/applications": {
+            "get": {
+                "tags": [
+                    "Applications"
+                ],
+                "description": "Lists all of the applications HDInsight cluster.",

```

Comment:
>Lists all of the applications HDInsight cluster. [](start = 32, length = 48)

Could you check this text string? It does not sound right.

Reply:
Fixed grammar.